Java/Java.sublime-syntax

```diff
@@ -1128,8 +1128,19 @@ contexts:
     - match: \n
       scope: invalid.illegal.newline.java
       pop: true
-    - match: \\.
+    - include: strings-escape
+
+  strings-escape:
+    - match: \\u+\h{4}
```

Comment:
Java allows at most 2 leadig `u`. So this should look like `\\uu?\h{4}`

Reply:
[The language specification](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.3) does not limit the amount of 'u' characters. The compiles also seems not to care about it.

Reply:
You're right, didn't read the BNF carefully enough.